src/kanata/mod.rs

```diff
@@ -2067,8 +2086,30 @@ fn apply_speed_modifiers() {
     assert_eq!(apply_mouse_distance_modifiers(10, &vec![33u16, 200u16]), 6);
 }
 
+#[cfg(feature = "passthru_ahk")]
+/// Clean kanata's state without exiting
+pub fn clean_state(kanata: &Arc<Mutex<Kanata>>, tick: u128) -> Result<()> {
+    let mut k = kanata.lock();
+    #[cfg(all(not(feature = "interception_driver"), target_os = "windows"))]
+    let layout = k.layout.bm();
+    #[cfg(all(not(feature = "interception_driver"), target_os = "windows"))]
+    release_normalkey_states(layout);
+    k.tick_ms(tick, &None)?;
+    #[cfg(not(target_os = "linux"))]
+    {
+        let mut k_pressed = PRESSED_KEYS.lock();
+        // debug!("  PRESSED {:?} tick {:?}", k_pressed, tick);
+        for key_os in k_pressed.clone() {
+            k.kbd_out.release_key(key_os)?;
+        }
+        k_pressed.clear();
+    }
+    Ok(())
+}
+
 /// Checks if kanata should exit based on the fixed key combination of:
 /// Lctl+Spc+Esc
+#[cfg(not(feature = "passthru_ahk"))]
```

Comment:
1. Why does this need to be compiled out for `passthru_ahk`?
2. If it truly needs to be commented out, you should instead compile out the body to not have to sprinkle more conditional compilation flags throughout the other files (i.e. unknown/macos/linux)